arbos/parse_l2.go

```diff
@@ -95,6 +107,8 @@ const (
 	L2MessageKind_Heartbeat          = 6 // deprecated
 	L2MessageKind_SignedCompressedTx = 7
 	// 8 is reserved for BLS signed batch
+	L2MessageKind_EspressoSequencedTx = 10
```

Comment:
Using the espresso sequencer as sequencer for the L2 (this is deprecated, in favor of the sovereign sequencer).